src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/_create.py

```diff
@@ -81,12 +106,14 @@ def _build_arguments_schema(cls, *args, **kwargs):
             options=["--ingestion-access"],
             arg_group="Properties",
             help="The public network access type to access workspace ingestion.",
+            default="Enabled",
```

Comment:
Is it a breaking change from None to Enabled

Reply:
it's enabled from old swagger definition, behavior are the same as before